<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgTransicionImagen</class>
 <widget class="QDialog" name="dlgTransicionImagen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transicion de dos Imagenes</string>
  </property>
  <widget class="QLabel" name="lblArchivo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona el Archivo de Imagen a Mostrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtArchivo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAbrirArchivo">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>40</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCara1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnProcesarImagen">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Procesar Imagen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNuevaImagen">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Nueva Imagen</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCara2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblMensaje">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblCara3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cboCara">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblCara">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Elige la Cara</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNroVeces">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nro Veces</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="txtNroVeces">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>4</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
